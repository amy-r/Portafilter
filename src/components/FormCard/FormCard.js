import React from 'react';
import '../Card/Card.css';
import PropTypes from 'prop-types';

export const FormCard = (props) => {
  return (
    <div className='roaster-card'>
      <h3 className='roaster-name'> { props.name } </h3>
      <ul id='roaster-list'>
        <li> Overall Score: { props.overallScore } </li>
        <li> Region: { props.region } </li>
        <li> Acidity: { props.acidity } </li>
        <li> Body: { props.body } </li>
        <li> Sweetness: { props.sweetness } </li>
        <li> Tactile: { props.tactile } </li>
      </ul>
    </div>
  );
};

FormCard.propTypes = {
  name: PropTypes.string,
  overallScore: PropTypes.string,
  region: PropTypes.string,
  acidity: PropTypes.string,
  body: PropTypes.string,
  sweetness: PropTypes.string,
  tactile: PropTypes.string
};  