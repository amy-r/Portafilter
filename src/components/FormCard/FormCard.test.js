import React from 'react';
import ReactDOM from 'react-dom';
import { FormCard } from './FormCard';
import { shallow } from 'enzyme'

describe('FormCard', () => {
  it('should exist', () => {
    const wrapper = shallow(<FormCard />)

    expect(wrapper).toMatchSnapshot();
  })
})