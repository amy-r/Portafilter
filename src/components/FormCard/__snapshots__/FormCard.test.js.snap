// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormCard should exist 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3
          className="roaster-name"
>
           
           
</h3>,
        <ul
          id="roaster-list"
>
          <li>
                     Overall Score: 
                     
          </li>
          <li>
                     Region: 
                     
          </li>
          <li>
                     Acidity: 
                     
          </li>
          <li>
                     Body: 
                     
          </li>
          <li>
                     Sweetness: 
                     
          </li>
          <li>
                     Tactile: 
                     
          </li>
</ul>,
      ],
      "className": "roaster-card",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            " ",
            undefined,
            " ",
          ],
          "className": "roaster-name",
        },
        "ref": null,
        "rendered": Array [
          " ",
          undefined,
          " ",
        ],
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
               Overall Score: 
               
</li>,
            <li>
               Region: 
               
</li>,
            <li>
               Acidity: 
               
</li>,
            <li>
               Body: 
               
</li>,
            <li>
               Sweetness: 
               
</li>,
            <li>
               Tactile: 
               
</li>,
          ],
          "id": "roaster-list",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " Overall Score: ",
                undefined,
                " ",
              ],
            },
            "ref": null,
            "rendered": Array [
              " Overall Score: ",
              undefined,
              " ",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " Region: ",
                undefined,
                " ",
              ],
            },
            "ref": null,
            "rendered": Array [
              " Region: ",
              undefined,
              " ",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " Acidity: ",
                undefined,
                " ",
              ],
            },
            "ref": null,
            "rendered": Array [
              " Acidity: ",
              undefined,
              " ",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " Body: ",
                undefined,
                " ",
              ],
            },
            "ref": null,
            "rendered": Array [
              " Body: ",
              undefined,
              " ",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " Sweetness: ",
                undefined,
                " ",
              ],
            },
            "ref": null,
            "rendered": Array [
              " Sweetness: ",
              undefined,
              " ",
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                " Tactile: ",
                undefined,
                " ",
              ],
            },
            "ref": null,
            "rendered": Array [
              " Tactile: ",
              undefined,
              " ",
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3
            className="roaster-name"
>
             
             
</h3>,
          <ul
            id="roaster-list"
>
            <li>
                         Overall Score: 
                         
            </li>
            <li>
                         Region: 
                         
            </li>
            <li>
                         Acidity: 
                         
            </li>
            <li>
                         Body: 
                         
            </li>
            <li>
                         Sweetness: 
                         
            </li>
            <li>
                         Tactile: 
                         
            </li>
</ul>,
        ],
        "className": "roaster-card",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              " ",
              undefined,
              " ",
            ],
            "className": "roaster-name",
          },
          "ref": null,
          "rendered": Array [
            " ",
            undefined,
            " ",
          ],
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                 Overall Score: 
                 
</li>,
              <li>
                 Region: 
                 
</li>,
              <li>
                 Acidity: 
                 
</li>,
              <li>
                 Body: 
                 
</li>,
              <li>
                 Sweetness: 
                 
</li>,
              <li>
                 Tactile: 
                 
</li>,
            ],
            "id": "roaster-list",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " Overall Score: ",
                  undefined,
                  " ",
                ],
              },
              "ref": null,
              "rendered": Array [
                " Overall Score: ",
                undefined,
                " ",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " Region: ",
                  undefined,
                  " ",
                ],
              },
              "ref": null,
              "rendered": Array [
                " Region: ",
                undefined,
                " ",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " Acidity: ",
                  undefined,
                  " ",
                ],
              },
              "ref": null,
              "rendered": Array [
                " Acidity: ",
                undefined,
                " ",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " Body: ",
                  undefined,
                  " ",
                ],
              },
              "ref": null,
              "rendered": Array [
                " Body: ",
                undefined,
                " ",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " Sweetness: ",
                  undefined,
                  " ",
                ],
              },
              "ref": null,
              "rendered": Array [
                " Sweetness: ",
                undefined,
                " ",
              ],
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  " Tactile: ",
                  undefined,
                  " ",
                ],
              },
              "ref": null,
              "rendered": Array [
                " Tactile: ",
                undefined,
                " ",
              ],
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
